marklogic-client-api/src/main/java/com/marklogic/client/expression/PlanBuilderBase.java

```diff
@@ -558,6 +590,8 @@ interface PlanBase {
      * Use AccessPlan as the type for instances of AccessPlan.
      */
     interface AccessPlanBase {
+        PlanBuilder.Plan bindParam(String param, AbstractWriteHandle content);
+        Map<String, AbstractWriteHandle> getContentParams();
```

Comment:
This is the mechanism for allowing a user to bind a param containing what I am calling "content", and I think an AbstractWriteHandle is the right choice here, as that gives the user plenty of flexibility without having to construct a DocumentWriteSet.